lib/scoring.rb

```diff
@@ -0,0 +1,59 @@
+module Scrabble
+
+  VALUES = {"a"=> 1, "b"=> 3, "c"=> 3, "d"=> 2, "e"=> 1, "f"=> 4, "g"=> 2, "h"=> 4, "i"=> 1, "j"=> 8, "k"=> 5, "l"=> 1, "m"=> 3, "n"=> 1, "o"=> 1, "p"=> 3, "q"=> 10, "r"=> 1, "s"=> 1, "t"=> 1, "u"=> 1, "v"=> 4, "w"=> 4, "x"=> 8, "y"=> 4, "z"=> 10}
+
+  class Scrabble
+
+    def self.score_of_each_letter(word)
+      score_by_letter = []
+      word.each_char do |letter|
+        VALUES.each do |key, value|
```

Comment:
This is pretty inefficient since you could potentially need to cycle through all 26 letters for each letter in the word. Can you see how to potentially make this more efficient?